serde_core/Cargo.toml

```diff
@@ -0,0 +1,56 @@
+[package]
+name = "serde_core"
+version = "1.0.188" # remember to update html_root_url and serde_derive dependency
+authors = ["Erick Tryzelaar <erick.tryzelaar@gmail.com>", "David Tolnay <dtolnay@gmail.com>"]
+build = "build.rs"
+categories = ["encoding", "no-std", "no-std::no-alloc"]
+description = "A generic serialization/deserialization framework"
+documentation = "https://docs.rs/serde"
```

Comment:
Does it make a difference whether this is serde or serde_core?

Reply:
I think this should point to `serde_core`. `serde` API matches that of `serde_core` verbatim, but it may not always be the case.